config/config.go

```diff
@@ -1132,7 +1132,7 @@ func getDefaultSubnetConfig(v *viper.Viper) subnets.Config {
 	return subnets.Config{
 		ConsensusParameters:         getConsensusConfig(v),
 		ValidatorOnly:               false,
-		ProposerMinBlockDelay:       proposervm.DefaultMinBlockDelay,
+		ProposerMinBlockDelay:       v.GetDuration(ProposerMinBlockDelayKey),
```

Comment:
This causes some weirdness in the defaults.

1. The default value of `ProposerMinBlockDelay` for all subnets is currently `proposervm.DefaultMinBlockDelay`.
2. With this change, the default value for subnets that do not specify a custom subnet config is still `proposervm.DefaultMinBlockDelay`.
3. With this change, the default value for subnets that do specify a custom subnet config is equal to `--proposer-min-block-delay`.

I think (2) and (3) should be unified.